src/parser/content-parser.php

```diff
@@ -14,6 +14,8 @@ class ContentParser {
 	protected $block_registry;
 	protected $post_id;
 	protected $warnings = [];
+	protected $blocks_to_include = [];
```

Comment:
I think we can skip adding these to the class because they're only used in `parse()`. That'll help keep the state a little easier to reason about.

Reply:
Yip, agreed. Removed